--- gnome-tour-42.0/src/meson.build.orig	2022-03-19 18:09:13.817518700 +0100
+++ gnome-tour-42.0/src/meson.build	2022-05-20 21:03:40.346043909 +0200
@@ -20,6 +20,7 @@ run_command(
 
 
 cargo_options = [ '--manifest-path', meson.project_source_root() / 'Cargo.toml' ]
+cargo_options += [ '--target', 'x86_64-unknown-linux-gnux32' ]
 cargo_options += [ '--target-dir', meson.project_build_root() / 'src' ]
 
 if get_option('profile') == 'default'
@@ -48,7 +49,7 @@ cargo_build = custom_target(
     cargo, 'build',
     cargo_options,
     '&&',
-    'cp', 'src' / rust_target / meson.project_name(), '@OUTPUT@',
+    'cp', 'src' / 'x86_64-unknown-linux-gnux32' / rust_target / meson.project_name(), '@OUTPUT@',
   ]
 )
 
